pandas/_libs/properties.pxd

```diff
@@ -0,0 +1,5 @@
+
+cdef class PropertyCacheMixin:
```

Comment:
mixing this in to other classes is going to cause many headaches, e.g. pickle and inability to inherit from other cython classes